src/idv_service/session/create/face_capture/create.face.capture.resource.payload.builder.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const CreateFaceCaptureResourcePayload = require('./create.face.capture.resource.payload');
+const Validation = require('../../../../yoti_common/validation');
+
+/**
+ *
+ * @class CreateFaceCaptureResourcePayloadBuilder
+ */
+class CreateFaceCaptureResourcePayloadBuilder {
+  /**
+   * Sets the id of the requirement that the resource will be used to satisfy.
+   *
+   * @param string requirementId
+   * @return CreateFaceCaptureResourcePayloadBuilder
+   */
+  withRequirementId(requirementId) {
+    Validation.isString(requirementId, 'requirement_id');
```

Comment:
In every builder that have been added in this PR, the patter for the validation is passing as last parameter the name of the field... so the any error message could include the name of the field.
That is quite useful when validating an object and its internal fields, less so when actually validating some parameters of a method... but for the little benefit it provides, I think we would need to re-use the same name as the parameter (see camel case vs snake case). I have check in the rest of the project, and the rules seems to use the name of the parameter, ie `requirementId` here.

Overall, it would be better to have a nicer message, which let you know that it failed with a parameter {insert name of parameter} for a given method... but maybe we will simply just change the code to improve validation another time so we would not do that.